src/main/java/org/openstreetmap/atlas/geography/PolyLine.java

```diff
@@ -204,6 +205,27 @@ public boolean addAll(final Collection<? extends Location> collection)
         return result;
     }
 
+    /**
+     * Append the given {@link PolyLine} to this one, if possible.
+     *
+     * @param other
+     *            The {@link PolyLine} to append
+     * @return the new, combined {@link PolyLine}
+     */
+    public PolyLine append(final PolyLine other)
```

Comment:
Good question. I thought it'd be easier to determine whether you can append/prepend right away and fail rather than having the client check if an append/prepend was done. Also, was trying to follow the pattern across this class (see get at index or offsetFromStart methods). Open to changing this if anyone feels strongly.